greenpithumb/io.py

```diff
@@ -0,0 +1,55 @@
+#import RPi.GPIO as GPIO
+
+# TODO(mtlynch): Move these pin constants to a configuration file so they're not
+# hardcoded in.
+
+# Raspberry Pi pin to which each component is connected
+PIN_PUMP = 0
+
+# Signal to turn a Raspberry Pi pin off.
+SIGNAL_OFF = 0
+
+# Signal to turn a Raspberry Pi pin on.
+SIGNAL_ON = 1
+
+
+class Error(Exception):
+    pass
+
+
+class WriteError(Error):
+    pass
+
+
+class IO(object):
+    """Wrapper for input and output on a Raspberry Pi board.
+
+    This wraps the board and allows the caller to read or send signals to
+    the Raspberry Pi's channels.
+    """
+
+    def __init__(self, placeholder_gpio):
+        """Creates a new input/output wrapper.
+
+        Args:
+            placeholder_gpio: Placeholder for RPi.GPIO.
+        """
+        self.GPIO = placeholder_gpio
+
+    def write_pin(self, channel, value):
```

Comment:
I think we should say `pin` instead of `channel`. The ADC's inputs are channels, but the GPIO pins are just pins.